bundles/org.openhab.binding.goecharger/src/main/java/org/openhab/binding/goecharger/internal/handler/GoEChargerHandler.java

```diff
@@ -384,8 +405,6 @@ private void sendData(String key, String value) {
     private GoEStatusResponseDTO getGoEData()
             throws InterruptedException, TimeoutException, ExecutionException, JsonSyntaxException {
         String urlStr = getUrl(GoEChargerBindingConstants.API_URL);
-        logger.debug("GET URL = {}", urlStr);
-
         ContentResponse contentResponse = httpClient.newRequest(urlStr).method(HttpMethod.GET)
                 .timeout(5, TimeUnit.SECONDS).send();
 
```

Comment:
I do not understand how it can work if you are not defining the new channels in the thing type XML file ?!